executor/src/main/java/com/redhat/service/bridge/executor/ExecutorsService.java

```diff
@@ -31,49 +29,37 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(ExecutorsService.class);
 
-    private Map<String, Set<Executor>> bridgeToProcessorMap = new HashMap<>();
-
     @Inject
-    ExecutorFactory executorFactory;
+    ExecutorConfigProvider executorConfigProvider;
 
     public void init(@Observes StartupEvent ev) {
         ExtensionProvider.getInstance().registerExtension(BridgeCloudEventExtension.class, BridgeCloudEventExtension::new);
     }
 
-    public void createExecutor(ProcessorDTO processorDTO) {
-
-        Executor executor = executorFactory.createExecutor(processorDTO);
-
-        synchronized (bridgeToProcessorMap) {
-            Set<Executor> executors = bridgeToProcessorMap.get(processorDTO.getBridge().getId());
-            if (executors == null) {
-                executors = new HashSet<>();
-            }
-
-            executors.add(executor);
-            bridgeToProcessorMap.put(processorDTO.getBridge().getId(), executors);
-        }
-    }
-
     @Incoming(EVENTS_IN_TOPIC)
-    public void processBridgeEvent(String event) {
-
+    public CompletionStage<Void> processBridgeEvent(final Message<String> message) {
         try {
-            CloudEvent cloudEvent = CloudEventUtils.decode(event);
+            CloudEvent cloudEvent = CloudEventUtils.decode(message.getPayload());
             BridgeCloudEventExtension bridgeCloudEventExtension = ExtensionProvider.getInstance().parseExtension(BridgeCloudEventExtension.class, cloudEvent);
-            Set<Executor> executors = bridgeToProcessorMap.get(bridgeCloudEventExtension.getBridgeId());
+            String bridgeId = bridgeCloudEventExtension.getBridgeId();
+            Set<Executor> executors = executorConfigProvider.getExecutors(bridgeId);
+            if (executors == null) {
+                LOG.info("[executor] A message for BridgeID {} has been received, but no executors were found.", bridgeId);
```

Comment:
Can this happen? I expect it should be impossible to deploy a runner without any executor so I would at least move log from `info` to `error`. Wdyt?

Reply:
@danielezonca at the moment, with the all-in-one application, it is possible: the user can 
1) create the Bridge instance
3) **NO PROCESSORS ARE CREATED FOR THE BRIDGE**
3) push events to the ingress
3) the all-in-one-executor will consume every message and fetch the Executors by Bridge ID -> if the user did not create any of them, we just ignore the message because no processors have been created. 

ATM, it is not a `warn` or an `error` because the user can upload events without having created a Processor. 
We might turn it into a `debug` message actually, but I think it's fine to have it as `info` in this early phase.